classtime.js

```diff
@@ -1,210 +1,321 @@
-var currentDay = 0;
-var currentHours = 0;
-var currentMinutes = 0; 
-var currentSeconds = 0;
-var currentDate;
-
-var currentClassPeriodIndex = -1;
-//nextClassPeriodIndex
-
-var currentScheduleIndex = -1;
-
-
-var use24HourTime = getLocalStorageBoolean("use24HourTime", false);
-
-var data = {
-    fullName: "",
-    shortName: "",
```

Comment:
why did the fullname and Shortname get removed? these were for storing information about the school so it could be displayed on the schedule page once we moved the data to a database and had multiple schools